spec/Section 3 -- Type System.md

```diff
@@ -1514,10 +1516,10 @@ type of an Object or Interface field.
 **Input Coercion**
 
 The value for an input object should be an input object literal or an unordered
-map supplied by a variable, otherwise a query error must be thrown. In either
+map supplied by a variable, otherwise a request error must be raised. In either
 case, the input object literal or unordered map must not contain any entries
-with names not defined by a field of this input object type, otherwise an error
-must be thrown.
+with names not defined by a field of this input object type, otherwise a
+response error must be raised.
```

Comment:
@benjie I noticed this term "response error" which is used only once in the spec and was introduced in this PR (which was the PR that introduced the term "request error"). Was this a mistake and should also say "request error"?

Reply:
Indeed, I suspect it should either be `request error` or `field error`. I'd have to read it carefully to figure out which - I'll add this to my list to review. Thanks for spotting!

Reply:
It should have been `request error` indeed. Thanks for spotting!